tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,798 @@
+to all NYC TLC taxi rides taken in January 2016. You can imagine that in a
+more expansive scenario, you will want to examine rides taken over several years.
+
+#### How many rides took place on each day?
+
+The first question you will explore is simple: *How many rides took place on each day during January 2016?*
+
+Since TimescaleDB supports full SQL, all that’s required is a simple SQL query 
+to count the number of rides and group/ order them by the day they took place, 
+as seen below:
+
+```sql
+-- What's the total number of rides that took place everyday for first 5 days
+SELECT date_trunc('day', pickup_datetime) as day, COUNT(*) FROM rides GROUP BY day ORDER BY day;
+```
+
+With this information, we know *how many* rides are taken each day, and we can
+identify the days of the week and month in which the most rides take place.
+Your result should look like this:
+
+```sql
+         day         | count  
+---------------------+--------
+ 2016-01-01 00:00:00 | 345037
+ 2016-01-02 00:00:00 | 312831
+ 2016-01-03 00:00:00 | 302878
+ 2016-01-04 00:00:00 | 316171
+ 2016-01-05 00:00:00 | 343251
+ 2016-01-06 00:00:00 | 348516
+ 2016-01-07 00:00:00 | 364894
+ 2016-01-08 00:00:00 | 392070
+ 2016-01-09 00:00:00 | 405825
+ 2016-01-10 00:00:00 | 351788
+ 2016-01-11 00:00:00 | 342651
+ 2016-01-12 00:00:00 | 367390
+ 2016-01-13 00:00:00 | 395090
+ 2016-01-14 00:00:00 | 396473
+ 2016-01-15 00:00:00 | 401289
+ 2016-01-16 00:00:00 | 411899
+ 2016-01-17 00:00:00 | 379156
+ 2016-01-18 00:00:00 | 341481
+ 2016-01-19 00:00:00 | 385187
+ 2016-01-20 00:00:00 | 382105
+ 2016-01-21 00:00:00 | 399654
+ 2016-01-22 00:00:00 | 420162
+ 2016-01-23 00:00:00 |  78133
+ 2016-01-24 00:00:00 | 159766
+ 2016-01-25 00:00:00 | 282087
+ 2016-01-26 00:00:00 | 327655
+ 2016-01-27 00:00:00 | 359180
+ 2016-01-28 00:00:00 | 383326
+ 2016-01-29 00:00:00 | 414039
+ 2016-01-30 00:00:00 | 435369
+ 2016-01-31 00:00:00 | 361505
+ 2017-11-17 00:00:00 |      2
+(32 rows)
+```
+
+#### What is the average fare amount for 2+ passengers?
+
+But this initial analysis is incomplete. We also need to understand how long each
+ride is. After all, if we're looking to lessen the impact on the environment, we
+probably want to discourage idle taxis and short trips.
+
+One way we can glean insight into the duration of trips is by looking into 
+the daily average fare amount for rides with 2+ passengers. Once 
+again, this is a simple four line SQL query with some conditional statements,
+shown in the query below:
+
+```sql
+-- What is the daily average fare amount for rides with 2 or more passengers
+-- for first 7 days?
+SELECT date_trunc('day', pickup_datetime) as day, avg(fare_amount) FROM rides WHERE passenger_count > 1 AND pickup_datetime < '2016-01-08' GROUP BY day ORDER BY day;
+```
+
+>:TIP: Queries like the ones above execute up to 20x faster on TimescaleDB than on a vanilla PostgreSQL database, thanks to Timescale’s automatic time and space partitioning.
```

Comment:
```suggestion
>:TIP: Queries like the ones above execute up to 20x faster on large datasets with TimescaleDB vs. a vanilla PostgreSQL database, thanks to TimescaleDB’s automatic time and space partitioning.
```
It's important to stress this b/c otherwise the common complaint is that benchmarks don't show a difference between the two